SIL.Core/IO/FileLocator.cs

```diff
@@ -309,7 +310,8 @@ public static string GetDirectoryDistributedWithApplication(bool optional, param
 			if (Directory.Exists(path))
 				return path;
 
-			foreach (var directoryHoldingFiles in new[] {"", "DistFiles", "common" /*for wesay*/, "src" /*for Bloom*/})
+			var directoriesHoldingFiles = new[] {"DistFiles", "common" /*for wesay*/, "src" /*for Bloom*/};
+			foreach (var directoryHoldingFiles in directoriesHoldingFiles)
```

Comment:
Note, this change removes the empty string from the list of subdirectories to try. Unless I'm misreading the code, having it caused a redundant repeat check of the root directory we just tried. Not having it also makes it easier to produce a nice message.